memory/memory/memory.py

```diff
@@ -0,0 +1,100 @@
+import aioredis
+import asyncio
+import base64
+import concurrent
+import json
+import logging
+import os
+import uvloop
+from aiohttp import web
+import kubernetes_asyncio as kube
+
+from hailtop.config import get_deploy_config
+from hailtop.google_storage import GCS
+from hailtop.hail_logging import AccessLogger
+from hailtop.tls import get_in_cluster_server_ssl_context
+from hailtop.utils import AsyncWorkerPool, retry_transient_errors
+from gear import setup_aiohttp_session, rest_authenticated_users_only
+
+uvloop.install()
+
+BATCH_PODS_NAMESPACE = os.environ['HAIL_BATCH_PODS_NAMESPACE']
+log = logging.getLogger('batch')
+routes = web.RouteTableDef()
+
+socket = '/tmp/redis.sock'
+
+@routes.get('/healthcheck')
+async def healthcheck(request):  # pylint: disable=unused-argument
+    return web.Response()
+
+@routes.post('/getfile')
+@rest_authenticated_users_only
+async def getfile(request, userdata):
+    body = await request.json()
+    billing_project = body['billing_project']
```

Comment:
Hmm.  This doesn't seem used now, let's remove it.  This is also tricky: I run pipelines under two billing accounts that use the same input file.  Do they share the cache?  Who pays for it?